vhdbuilder/packer/test/linux-vhd-content-test.sh

```diff
@@ -235,29 +235,30 @@ testKubeProxyImagesPulled() {
         "1.18.10-hotfix.20210310.2",
         "1.18.14-hotfix.20210511",
         "1.18.14-hotfix.20210525",
-        "1.18.17-hotfix.20210525",
         "1.18.17-hotfix.20210525.1",
-        "1.18.19-hotfix.20210522",
+        "1.18.17-hotfix.20210525.2"
```

Comment:
please add a comma after 1.18.17-hotfix.20210525.2
the missing command had caused the tests to fail earlier too. thanks.